10up/simple-podcasting
# feat/40: Add onboarding wizard (phase -1)

Closes #40 


### Verification Process

1. Delete all podcasts from the local setup
2. Deactivate the plugin
3. Activate the plugin
4. Observe the setup wizard.

### Screenshots

### Step 1:

<img width="1792" alt="Screenshot 2022-10-26 at 9 13 57 PM" src="https://user-images.githubusercontent.com/17757960/198072853-d15a20cd-53e4-47f7-bfb3-1a6c875fb6fd.png">

### Step 2:

<img width="1792" alt="Screenshot 2022-10-26 at 9 14 12 PM" src="https://user-images.githubusercontent.com/17757960/198072927-b407ffed-c6c8-4dc2-bf80-3153548bcd88.png">

### Checklist:






### Changelog Entry



### Credits


Props @helen @jeffpaul @Nicolas-knight @jnetek @Sidsector9 @cadic
